sopel/bot.py

```diff
@@ -648,3 +646,71 @@ def cap_req(self, module_name, capability, arg=None, failure_callback=None,
             entry.append(_CapReq(prefix, module_name, failure_callback, arg,
                                  success_callback))
             self._cap_reqs[cap] = entry
+
+    def register_url_callback(self, pattern, callback):
+        """Register a ``callback`` for URLs matching the regex ``pattern``
+
+        :param pattern: compiled regex pattern to register
+        :param callback: callable object to handle matching URLs
+
+        .. versionadded: 7.0.0
```

Comment:
This directive won't appear in Sphinx's output as written; it needs two colons. (Also applies to the other two occurrences.)

I would also just put `7.0`, since we're not _ever_ (if I have anything to say about it) going to add/change/remove API methods in a patch release.

```suggestion
        .. versionadded:: 7.0
```

Reply:
Fixed.